shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -1335,6 +1337,30 @@ - (void)testItReportsHighContrastWhenTraitCollectionRequestsIt {
   [mockTraitCollection stopMocking];
 }
 
+- (void)testItReportsAlwaysUsed24HourClock {
+  // Setup test.
+  id settingsChannel = OCMClassMock([FlutterBasicMessageChannel class]);
+  OCMStub([self.mockEngine settingsChannel]).andReturn(settingsChannel);
+
+  id mockHourFormat = OCMClassMock([FlutterHourFormat class]);
+  OCMStub([mockHourFormat isAlwaysUse24HourFormat]).andReturn(YES);
+
+  FlutterViewController* vc = [[FlutterViewController alloc] initWithEngine:self.mockEngine
+                                                                    nibName:nil
+                                                                     bundle:nil];
+  // Exercise behavior under test.
+  [vc traitCollectionDidChange:nil];
+
+  // Verify behavior.
+  OCMVerify([settingsChannel sendMessage:[OCMArg checkWithBlock:^BOOL(id message) {
+                               return message[@"alwaysUse24HourFormat"];
+                             }]]);
+
+  // Clean up mocks
```

Comment:
```suggestion
  // Clean up mocks.
```

This appears to be an issue with existing tests above, but we can fix for this patch at least. :)